### DISCUSSION TITLE
iLogtail1.8版本的windows正式版什么时候能发布

### DISCUSSION THREAD

**Discussion by pigwantacat** [alibaba/ilogtail#1316] [Help] [answered]
在文档中找到了1.8的windows正式版，但是目前好像不支持下载
![1704697836049](https://github.com/alibaba/ilogtail/assets/79680042/8966ec47-d2af-4fa7-a22d-6cf13ffe02b1)

**Comment by messixukejia** [comment, marked as answer]
Windows不是每个版本都发布的。可以先用之前的版本吧。
